### ISSUE TITLE
args for training maps?

### ISSUE THREAD

**Issue by veroveroxie** [ermongroup/alignflow#3]
Hi, Thanks a lot for sharing your code!

could you please share the args for training Maps dataset? 

I have tried with resize_shape='96,96', crop_shape="64,64", lambda_mle=1e-5, but I can only get MSE_src2tgt=0.024, MSE_tgt2src=0.161, is there something wrong?

BTW,  how do you evaluate? I noticed that you used nn.BatchNorm2d instead of BatchNormStats,  did you use the running stats or use the input batch stats?
